backend/app/api/schemas.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Filter(str, Enum):
+    free = 'free'
+
+
+class Sender(str, Enum):
+    volunteer = 'volunteer'
+    client = 'client'
+
+
+class MessageBase(BaseModel):
+    created_at: datetime
+    content: str
+    sender: Sender
+
+
+class MessageCreate(MessageBase):
+    pass
+
+
+class Message(MessageBase):
+    pass
+
+
+class SupportBase(BaseModel):
+    question: str
+    client_id: Optional[int]
+    volunteer_id: Optional[int]
+    messages: Optional[list[Message]] = []
+
+    class Config:
+        orm_mode = True
+
+
+class SupportCreate(SupportBase):
+    id: Optional[int]
+
+
+class SupportThread(SupportBase):
+    id: int
+
+
+class VolunteerBase(BaseModel):
+    tg_id: int
+    thread_id: Optional[int]
```

Comment:
This is a data duplication: threads store which volunteer is assigned to them and volunteers store which thread they are assigned to. This must not be the case for the database schema, and probably this should not either be the case for the schemas here, should it?

Let me know if you think we should fix this on the database attachment stage.

Reply:
Besides, is it ever possible to create a volunteer with `thread_id` set at the moment of creation? Because this is what the current schema `VolunteerCreate` allows to do (and what backend/app/api/services/volunteer_service.py would happily process...)

Reply:
I receive a VolunteerCreate object in PUT handle
`@router.put('/volunteers/{volunteer_id}', response_model=Volunteer)
async def update_volunteer(volunteer_id: int, volunteer: VolunteerCreate)`

Reply:
I'm not sure what this comment refers to. Is this about volunteer creation? If so, I'm telling you I think this is incorrect: when a new volunteer is created, they are never assigned to any thread at the moment of creation. So, it's incorrect that the method `PUT /volunteers/<id>` optionally accepts the `thread_id` parameter